cookbooks/multicomponent_steinberger/doc/multicomponent_steinberger.md

```diff
@@ -0,0 +1,148 @@
+```
+Since we need to use the heat capacity, density, and thermal conductivity to
+calculate the thermal diffusivity (function constant `kappa`) and NONE of these values is constant, you should choose values that are representative for the model's bottom boundary layer. You can do it by first setting `End time = 1`
+and running the model. You then find the representative heat capacity, density, and thermal conductivity values for your model from the visualization output. The heat capacity and density are obtained from the look-up tables, whereas the thermal conductivity is pressure-temperature dependent, which is set in the `material model`. 
+```{literalinclude} conductivity.setup.prm
+```
+Other than `kappa`, other constants used in the `Function`, the adiabatic bottom temperature (`Tab`)and the age of the bottom thermal boundary layer (`age`), should also be consistent with the adiabatic temperature setup of the model.
+
+You may also notice that there is a `subsection Function` under `subsection Adiabatic`, which has the same expression as the basal layer prescribed in `Initial composition model/Function`. Because the adiabat of the ambient mantle is computed using the thermal properties in the two look-up tables based on the basalt fraction, by using the same function expression, we could compute an initial mantle adiabat that is consistent with the chemical composition. If the `Function express` is set to `0`, the mantle adiabat will be computed assuming the composition is  100% of the background materials (pyrolite in our case). This function expression should have the same number of terms as the `Number of fields` set in the
+`Compositional fields`. Please see [ASPECT forum discussion](https://community.geodynamics.org/t/there-is-a-problem-when-use-initial-temperature-model-adiabatic-half-space-cooling-with-number-of-composition-greater-than-1-ascii-model/3160/2) for more information.
+
+
+The complete input file can be found in
+[cookbooks/multicomponent_steinberger/doc/steinberger_thermochemical_plume.prm](https://www.github.com/geodynamics/aspect/blob/main/cookbooks/multicomponent_steinberger/doc/steinberger_thermochemical_plume.prm).
+
+## Results
+
+We run the model for 300 million years. A primary plume rise from the initial thermochemical anomaly to the base of the lithosphere after ~250 Myrs. The thermochemical plume penetrates the 410 km buoyancy barrier broadly and spreading out at ~410 km depth. Our model results are consistent with the results from {cite}`Dannberg:Sobolev:2015`. However, due to the mid-mantle viscosity hump ({numref}`fig:viscosity.multicompstn`) presents in our model but not in {cite:t}`Dannberg:Sobolev:2015`, the radius of our plume rises much slower than the plumes in {cite:t}`Dannberg:Sobolev:2015`. You may need to adjust the reference
+viscosity profile and/or increase the buoyancy flux of the initial thermochemical anomaly to get similar results from {cite:t}`Dannberg:Sobolev:2015`.
```

Comment:
I would also add some additional text here that says something like:

"Also note that the cold downwellings that develop in this model push the initial dense layer at the base of the mantle into a pile of basaltic material. Since the existence of such a pile affects the formation of plumes, the first plume that rises represents a transient state of the model, and plumes that rise at later points in time might look different. For a research application model, one would want to run the model for a longer time and base interpretations on the dynamics of plumes after this initial transient phase."

I think your setup is great to teach a user how to use all of the features in ASPECT to make a model of a thermochemical plume, but we want to make sure that users know what the limitations are.

Reply:
Thanks for rewording! I meant to say increasing initial volume and excess temperature. I have added the discussion about the limitations of this model setup.